java/ql/lib/ext/javax.servlet.http.model.yml

```diff
@@ -26,6 +26,8 @@ extensions:
       - ["javax.servlet.http", "HttpServletResponse", False, "addHeader", "", "", "Argument[0..1]", "response-splitting", "manual"]
       - ["javax.servlet.http", "HttpServletResponse", False, "sendError", "(int,String)", "", "Argument[1]", "information-leak", "manual"]
       - ["javax.servlet.http", "HttpServletResponse", False, "setHeader", "", "", "Argument[0..1]", "response-splitting", "manual"]
+      - ["javax.servlet.http", "HttpSession", True, "putValue", "", "", "Argument[0..1]", "trust-boundary", "manual"]
+      - ["javax.servlet.http", "HttpSession", True, "setAttribute", "", "", "Argument[0..1]", "trust-boundary", "manual"]
```

Comment:
I think you're marking argument 0 (i.e. the name of the attribute) as a sink as well because of the risk of an attacker overriding more important session attributes. Is that correct?

Reply:
Yes. If user input can flow to the attribute name, you run the risk of the user controlling arbitrary attributes in the session.